.eslintrc

```diff
@@ -1,22 +1,36 @@
 {
+  "parser": "babel-eslint",
+  "extends": "airbnb",
   "env": {
     "browser": true,
     "node": true
   },
-  ecmaFeatures: {
-    jsx: true
-  },
-  "globals": {
-    "window": true,
-    "document": true
+  "rules": {
+    "arrow-parens": 0,
+    "class-methods-use-this": 0,          # HACK: Classes will be replaced by pure fn's
+    "comma-dangle": ["error", "never"],
```

Comment:
I suggest `"comma-dangle": 0`.

The trailing commas are good for VCS such as Github.